onedal/_device_offload.py

```diff
@@ -178,11 +178,23 @@ def _convert_to_dpnp(array):
 
 
 def support_usm_ndarray(freefunc=False, queue_param=True):
+    """
+    Handles USMArray input. Puts SYCLQueue from data to method arguments.
+
+    Parameters
+    ----------
+    freefunc (bool) : Set to True if decorates free function.
+    queue_param (bool) : Set to True if queue from data should be provided to the method args.
+                         Queue will not be changed if provided explicitly.
```

Comment:
```suggestion
    queue_param (bool) : Set to True if decorated function has a `queue` parameter.
                         Queue will not be changed if provided explicitly.
```

Reply:
I would say that my comment makes more sense. It might be possible that decorated function has `queue` parameter but user doesn't want to put queue from data there.

Reply:
outdated